taxonomies/categories.txt

```diff
@@ -69179,7 +69175,7 @@ sv:Rårörda, rårörd
 sv:Rårörda lingon
 
 <en:Fruit and vegetable preserves
-en:Jams
+en:Jams, fruit jams
```

Comment:
"fruit jams" is already defined as a child of "jams".

Reply:
I solved the issue